include/json/value.h

```diff
@@ -89,9 +76,17 @@ class JSON_API LogicError : public Exception {
 #endif
 
 /// used internally
-JSONCPP_NORETURN void throwRuntimeError(String const& msg);
-/// used internally
-JSONCPP_NORETURN void throwLogicError(String const& msg);
+#if JSON_USE_EXCEPTION
```

Comment:
Should you avoid double declarations by guarding with:

#if !defined(THROW_RUNTIME_ERROR) && !defined(THROW_LOGIC_ERROR)